### PR TITLE
chore(website): add multiple version api doc

### PR DESCRIPTION
**Pull request by PainterPuppets**
image

### CODE REVIEW

File: website/versioning-branches.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+const branches = [
+  '0.19',
+  'next/v0.19.0-alpha.0',
+  {
+    branchName: 'ckb2021',
+    label: 'CKB-2021'
+  }
+];
+
+const parseURL = (branchName) => branchName
```
Comment (zhangyouxin) [inline website/versioning-branches.js:13]: optional: lines 12-13 could be merged into one regex `[_\.]`

**Review by zhangyouxin — APPROVED** [review]
LGTM

File: website/versioning-branches.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+// string or { branchName: string, label: string }
+const branches = [
```
Comment (PainterPuppets) [inline website/versioning-branches.js:2]: just sample

File: website/versioning-branches.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  '0.19',
+  'next/v0.19.0-alpha.0',
+  {
+    branchName: 'ckb2021',
+    label: 'CKB-2021'
+  }
```
Comment (homura) [inline website/versioning-branches.js]:
```suggestion
  {
    branchName: 'stable',
    label: '0.18.0(latest)'
  },
  {
    branchName: 'develop',
    label: '0.19.x(next)'
  }
```

File: website/docusaurus.config.js [hunk 4]
```diff
@@ -82,10 +83,17 @@ const config = {
...
-            href: "/api/",
             label: "API",
             position: "left",
             target: "_blank",
+            items: [
+              {
+                href: "/api/",
+                label: "latest",
+                target: "_blank",
+              },
```
Comment (homura) [inline website/docusaurus.config.js]:
```suggestion
```

**Pull request merged**